stdlib/public/core/Filter.swift.gyb

```diff
@@ -268,6 +268,22 @@ public struct ${Self}<
     return ${Slice}(base: self, bounds: bounds)
   }
 
+  // Any estimate of the number of elements that pass `_predicate` requires
+  // iterating the collection and evaluating each element, which can be costly,
+  // is unexpected, and usually doesn't pay for itself in saving time through
+  // preventing intermediate reallocations. (SR-4164)
+  public var underestimatedCount: Int { return 0 }
+
+  public func _copyToContiguousArray()
```

Comment:
I'm not sure how it would be different from `_copySequenceToContiguousArray` then—without being able to rely on a truthful `count` to preallocate, it would have to do the same dance of allocating the estimate and then iterating through the elements.